container/src/lib.rs

```diff
@@ -226,24 +231,18 @@ impl<T: Clone + 'static> PushContainer for Vec<T> {
     }
 }
 
-impl<T> PushInto<Vec<T>> for T {
+impl<T> PushInto<T> for Vec<T> {
     #[inline]
-    fn push_into(self, target: &mut Vec<T>) {
-        target.push(self)
+    fn push_into(&mut self, item: T) {
+        self.push(item)
     }
 }
 
-impl<T: Clone> PushInto<Vec<T>> for &T {
-    #[inline]
-    fn push_into(self, target: &mut Vec<T>) {
-        target.push(self.clone())
-    }
-}
 
-impl<T: Clone> PushInto<Vec<T>> for &&T {
+impl<T: Clone> PushInto<&T> for Vec<T> {
     #[inline]
-    fn push_into(self, target: &mut Vec<T>) {
-        (*self).push_into(target);
+    fn push_into(&mut self, item: &T) {
+        self.push(item.clone())
     }
 }
```

Comment:
It looks like you lost the `PushInto<&&T>` implementation. If it's not needed, no worries, but wanted to check!